Quark_Sketch/src/arduino/serial_print.c

```diff
@@ -0,0 +1,123 @@
+#include "serial_print.h"
+#include "arduino.h"
+
+
+#include <zephyr.h>
+#include "curie_shared_mem.h"
+
+#if defined(CONFIG_STDOUT_CONSOLE)
+#include <stdio.h>
+#define PRINT           printf
+#else
+#include <misc/printk.h>
+#define PRINT           printk
+#endif
+
+#define Rx_BUFF curie_shared_data->cdc_acm_shared_rx_buffer.data
```

Comment:
I remember seeing these same defines in another file. Can they go in a common file? "curie_shared_mem.h" maybe?